src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Figure implements CalculateArea {
+    public static final String defaultColor = "WHITE";
+    private String color;
+
+    public void drawArea() {
+
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
```

Comment:
The `getArea()` method returns 0, which might not be meaningful for a base class. Consider making `Figure` abstract and `getArea()` an abstract method if subclasses are supposed to provide specific implementations.